examples/multisig.js

```diff
@@ -0,0 +1,17 @@
+const steem = require('../lib');
+
+const privWif1 = '5K2LA2ucS8b1GuFvVgZK6itKNE6fFMbDMX4GDtNHiczJESLGRd8';
+const privWif2 = '5JRaypasxMx1L97ZUX7YuC5Psb5EAbF821kkAGtBj7xCJFQcbLg';
```

Comment:
Yes they are, it doesn't matter as it's just posting private WIF of disposable accounts, having them hardcoded make easier for dev to test multi-sig.